### PR TITLE
docs: user: Just quick manual inspection

### PR DESCRIPTION
**Pull request by smoe**
No drama.

### CODE REVIEW

**Review by silopolis — APPROVED** [review]

File: docs/src/user/starting-linuxcnc.adoc [hunk 1]
```diff
@@ -35,19 +35,19 @@ Options:
...
             specify custom trajectory_planning_module
-            overrides optional ini setting [TRAJ]TPMOD
+            overrides optional INI setting [TRAJ]TPMOD
     -m "homemodulename [parameters]"
             specify custom homing_module
-            overrides optional ini setting [EMCMOT]HOMEMOD
+            overrides optional INI setting [EMCMOT]HOMEMOD
     -H "dirname": search dirname for Halfiles before searching
-                  ini directory and system library:
+                  INI directory and system library:
```
Comment (hansu) [inline docs/src/user/starting-linuxcnc.adoc:43]: This is a 1:1 printout of the command `linuxcnc -h`, so I would not change this.

Comment (smoe) [reply docs/src/user/starting-linuxcnc.adoc:43]: No, please accept and we also fix the `linuxcnc -h` accordingly.

Comment (smoe) [reply docs/src/user/starting-linuxcnc.adoc:43]: There are quite a lot of "ini"s in the source tree. I already started and will prepare a separate PR for that - am not sure that we want to change them all.

Comment (hansu) [reply docs/src/user/starting-linuxcnc.adoc]: Yeah but we shouldn't merge things that we know that they need to be changed later...

**Comment by silopolis** [comment]
Le dim. 4 sept. 2022 à 20:55, Hans Unzner ***@***.***> a
écrit :

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In docs/src/user/starting-linuxcnc.adoc
> <https://github.com/LinuxCNC/linuxcnc/pull/1994#discussion_r962354930>:
>
> > +            overrides optional INI setting [TRAJ]TPMOD
>      -m "homemodulename [parameters]"
>              specify custom homing_module
> -            overrides optional ini setting [EMCMOT]HOMEMOD
> +            overrides optional INI setting [EMCMOT]HOMEMOD
>      -H "dirname": search dirname for Halfiles before searching
> -                  ini directory and system library:
> +                  INI directory and system library:
>
> This is a 1:1 printout of the command linuxcnc -h, so I would not change
> this.
>
Then may be better to fix the script at the same time !?

**Pull request merged**